File: 1323-maximum-69-number/1323-maximum-69-number.cpp
#include<bits/stdc++.h>
class Solution {
public:
    int maximum69Number (int num) {
        bool stopped = false;
        string res;
        string numS = to_string(num);
        for(int i=0;i<numS.size();i++) {
            if(numS[i]=='6' && stopped==false) {
                res+='9';
                stopped=true;
            }
            else res+=numS[i];
        }
        return stoi(res);
    }
};

File: 1323-maximum-69-number/1323-maximum-69-number.py
class Solution:
    def maximum69Number (self, num: int) -> int:
        numS = str(num)
        res=""
        stopped = False
        for i in range(len(numS)):
            if(numS[i]=='6' and stopped==False):
                res+='9'
                stopped=True
            else: res+=numS[i]
        return int(res)